uk.co.compucorp.civicrm.hrleaveandabsences/xml/schema/CRM/HRLeaveAndAbsences/PublicHoliday.entityType.php

```diff
@@ -0,0 +1,11 @@
+<?php
+// This file declares a new entity type. For more details, see "hook_civicrm_entityTypes" at:
+// http://wiki.civicrm.org/confluence/display/CRMDOC/Hook+Reference
+return array (
+  0 => 
+  array (
+    'name' => 'PublicHoliday',
+    'class' => 'CRM_HRLeaveAndAbsences_DAO_PublicHoliday',
+    'table' => 'civicrm_hrleaveandabsences_public_holiday',
+  ),
+);
```

Comment:
You also need to add the entity to the **hrleaveandabsences_civicrm_entityTypes** hook in **hrleaveandabsences.php**, otherwise you will get some errors trying to use the API.